app/src/main/kotlin/com/x8bit/bitwarden/ui/vault/feature/migratetomyitems/MigrateToMyItemsViewModel.kt

```diff
@@ -23,13 +25,22 @@ private const val KEY_STATE = "state"
  */
 @HiltViewModel
 class MigrateToMyItemsViewModel @Inject constructor(
+    private val authRepository: AuthRepository,
+    private val policyManager: PolicyManager,
     savedStateHandle: SavedStateHandle,
 ) : BaseViewModel<MigrateToMyItemsState, MigrateToMyItemsEvent, MigrateToMyItemsAction>(
     initialState = savedStateHandle[KEY_STATE] ?: run {
-        val args = savedStateHandle.toMigrateToMyItemsArgs()
+
+        val orgId = policyManager.getPersonalOwnershipPolicyOrganizationId()
+        val orgName = authRepository.userStateFlow.value
+            ?.activeAccount
+            ?.organizations
+            ?.firstOrNull { it.id == orgId }
+            ?.name
```

Comment:
❓ Passing args makes for a simpler ViewModel; specifically in regards to handling scenarios when either `orgId` or `orgName` are null. Is there a reason why this approach is taken instead?

Reply:
I have changed this quite a bit from original idea. `RootNavViewModel` only verifies if it should navigate from the `shouldMigratePersonalVaultFlow`, not having any info regarding the org that should do the migration.

This was changed because if the vault was updated on a different client, the migration flow would still be displayed as it would not check with the most synced data. Now we are syncing before displaying the screen to make sure we are not displaying the screen unless it is needed.

This being said, I agree that if we could pass the parameters would simplify the VM, not sure about a good way to achieve it, only that I can think about is the flow having the Organization object?